### ISSUE TITLE
aarch64 used X86_64 platform

### ISSUE THREAD

**Issue by laoshanxi** [uraimo/run-on-arch-action#113]
I am currently using this to run arm64 docker container, but seems I can not get a QMENU ARM64 environment, still X86 runner, is there anything wrong with my configuration?

workflow:
https://github.com/laoshanxi/app-mesh/blob/main/.github/workflows/publish-github-release.yml#L59

action output:
https://github.com/laoshanxi/app-mesh/actions/runs/4329800139/jobs/7560595971
```
      - name: build with ubuntu18 ARM64
        uses: uraimo/run-on-arch-action@v2.5.0
        with:
          arch: aarch64
          distro: ubuntu_latest
          githubToken: ${{ github.token }}
          dockerRunArgs: |
            --volume ${{ github.workspace }}:/workspace --workdir /workspace
          base_image: laoshanxi/appmesh:build_ubuntu18
          shell: /bin/sh
          run: |
            mkdir ubuntu18arm
            cd ubuntu18arm
            cmake ..
            make -j4
            make pack
            ls
            make test ARGS='-V'
```

<img width="662" alt="image" src="https://user-images.githubusercontent.com/50568160/222881649-d167b76e-dbde-43e2-b6c2-13e6dd8b5435.png">

**Comment by uraimo** [comment]
Hi, yes, the problem is that the image is multi-platform and the action doesn't allow to explicitly specify the architecture. Adding support for --platform could be useful also for images that don't specify the architecture, but in this release I don't plan to add any more parameters.

AFAIK, you could select a specific version of the image adding the digest like `<username>/<image>:latest@sha256:<digest>` but since the digest changes you'll need to retrieve it with docker inspect every time and pass it to the run-on-arch job. It seems like it should work but I'm not sure if it's doable.

**Comment by laoshanxi** [comment]
will we plan support apply parameter `arch: aarch64` when we specified `base_image`?

**Comment by uraimo** [comment]
Yes, I will give a better name to the parameters but that's the idea, if you don't use the default images you can optionally specify a platform for the base_image.

**Comment by grimbough** [comment]
I think you can do this by specifying `--platform=linux/arm64` in the base_image argument e.g.

```
base_image: --platform=linux/arm64 bioconductor/bioconductor_docker:RELEASE_3_16
```

**Comment by laoshanxi** [comment]
Thanks, I found this way also works:
https://github.com/laoshanxi/app-mesh/blob/main/.github/workflows/publish-github-release.yml#L59
